scripts/build-win.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const tp = require('teen_process');
+const path = require('path');
+
+const APKS_DIR = './apks';
+const APK_FILENAME = '/ApiDemos-debug.apk';
+
+async function runGradleBuild() {
+  try {
+    await tp.exec('cmd.exe', ['/c', 'gradlew.bat', 'clean', 'assembleDebug']);
+  } catch (e) {
+    throw new Error('Failed at gradle build!\n ' + e);
+  }
+}
+
+async function copyAPK() {
+  try {
+    await fs.promises.access(APKS_DIR, fs.constants.R_OK);
+  } catch (e) {
+    await fs.promises.mkdir(APKS_DIR, { recursive: true });
+  }
+  await fs.promises.copyFile('app/build/outputs/apk/debug/app-debug.apk', path.join('apks', APK_FILENAME));
+}
+
+async function main() {
+  await runGradleBuild();
+  await copyAPK();
+}
+
+(async () => await main())();
```

Comment:
after all fixes the script will be cross-platform, so we could also simplify package tasks to only run this script for all platforms and rename the script itself to `build.js`

Reply:
Will do this once this one is merged since it's got too many changes already

Reply:
You can squash and rebase your changes to reduce the amount of commits. I would prefer to keep a single build task rather than adding a new one and then removing it

Reply:
ok. I will do that.

Reply:
Fixed the build by adding a validation on Operating System, so will decide which Gradlew command to use since win32 and Linux has diff commands